metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery.py

```diff
@@ -1381,3 +1383,21 @@ def add_config_to_report(self):
             self.config.start_time,
             self.config.end_time,
         )
+
+    def unquote_and_decode_unicode_escape_seq(
```

Comment:
Please see this https://datahubproject.io/docs/next/metadata-ingestion/developing/#code-style

Can this be a standalone helper method with unit tests? If yes, please change